Added workflow URLScan with Microsoft Teams (#135)
rapid7/insightconnect-workflows: workflows/URLScan_with_urlscanio_from_Microsoft_Teams/help.md

# Description

> This workflow looks up a URL given via Slack command in urlscan.io and reports information on the URL back to Microsoft Teams.

Reviewer: ```suggestion
Lookup and enrich suspicious URLs and domains with a simple Microsoft Teams message. URL and domain analysis provided by URLScan.io is returned in a Microsoft Teams message.
```

Sample Teams Trigger Commands:

> `!urlscan https://bit.ly/39HoYMu`

Reviewer 2: ```suggestion
`!enrich-url https://bit.ly/39HoYMu`
```

Reviewer 2: Change the Teams command to this per the new convention document

Author: I have updated the ICON file

# Key Features

> * Lookup URL from Microsoft Teams

Reviewer 2: ```suggestion
* Enrich URL from Microsoft Teams
```

> * Lookup URL from Microsoft Teams

Reviewer: ```suggestion
* **Investigate Indicators at Scale** - Manually investigating every reported phishing attempt is extremely difficult to scale. Automatic analysis of common phishing IOCs reduces the overhead associated with every reported incident.
* **Offload Repetitive Tasks** - Parsing out, analyzing, and enriching URLs manually can be repetitive and mind-numbing. Give that time back to your team so you can elevate your focus to identifying and resolving threats.
* **Shorten the Investigation Timeline** - Manage timely responses to real attacks by eliminating the investigation of false positives and spam. If the URLs are found to be malicious, pivot seamlessly to incident response.```

# Requirements

> * Microsoft Teams Connection

Reviewer: ```suggestion
* [Microsoft Teams](https://insightconnect.help.rapid7.com/docs/microsoft-teams)
```

* URLScan.io API Key

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

Once the workflow has been imported, each Microsoft Teams step will need the team name and channel name updated to suit your Teams environment (edit the input with the preset text of `change_me`).
The following are the 5 Microsoft Teams steps:

> - _!urlscan Trigger_

Reviewer: ```suggestion
- _!enrich-url Trigger_
```

- _Print No URL Found_
- _Print Failed to get Scan ID_
- _Print Failed to get Report_
- _Print URLScan Report_

> To run the workflow, send the command `!urlscan <URL>` in your configured team and channel. The workflow will post after it has completed.

Reviewer: ```suggestion
# Usage

*This workflow will only trigger in the channel specified in the Microsoft Teams workflow steps.*

To run the workflow, send a message to the specified Microsoft Teams channel starting with the command `!enrich-url`.

For example:

* `!enrich-url badsite.com`
* `!enrich-url http://another.badsite.com`
```
